src/pages/library/DataBrowserDetails.js

```diff
@@ -109,8 +116,43 @@ export const SidebarComponent = ({ dataObj, id }) => {
   const { access } = dataObj
   const [showRequestAccessModal, setShowRequestAccessModal] = useState(false)
   const [feedbackShowing, setFeedbackShowing] = useState(false)
+  const [datasetNotSupportedForExport, setDatasetNotSupportedForExport] = useState(false)
+  const [snapshotExportJobId, setSnapshotExportJobId] = useState()
+  const [tdrSnapshotPreparePolling, setTdrSnapshotPreparePolling] = useState(false)
   const sidebarButtonWidth = 230
 
+
+  const importDataToWorkspace = dataset => {
+    Utils.cond(
+      [isWorkspace(dataset), () => {
+        Ajax().Metrics.captureEvent(`${Events.catalogWorkspaceLink}:detailsView`, {
+          id,
+          title: dataObj['dct:title'],
```

Comment:
I don't think so, but I think in this case we want it, because product would like to know which datasets people are most interested in. (Although id could give us that, its not readable for product, so they would need to ask us for it.)

Reply:
Is the mix panel UI extensible? The title for an ID could be extracted pragmatically at display time by using the catalog `datasets` endpoint.

Reply:
I don't believe it is - and if it is, it would mean giving admin credentials to mixpanel? Otherwise we couldn't guarantee access?

Reply:
Right now catalog entries are publicly discoverable, so any user can get this metadata. Conversely, if a dataset _isn't_ publicly discoverable, maybe its metadata shouldn't be stored in mix panel?

Overall I have a preference here, to only send `id`, but isn't not a strong preference and am OK with sending more data too. I want to explore the impact / possibility of not sending other data as well, but I don't want to hold up the PR on this point.